index.html

```diff
@@ -45,9 +45,10 @@ <h1>Whats The Weather Today?</h1>
                 <input id="dallas" type="button" class="button" onclick="indexDallas.html" value="Go to Dallas Hub" />
             
         </div>
-
+    <article hidden id="displaycontent">
```

Comment:
Given you already have `#searchOutputData` can you not just apply/remove the `hidden` tag from that?